cadl-tests/src/test/java/com/models/property/optional/CollectionsModelClientTest.java

```diff
@@ -13,7 +13,7 @@
 
 class CollectionsModelClientTest {
 
-    CollectionsModelClient client = new ModelsPropertyOptionalClientBuilder().buildCollectionsModelClient();
+    CollectionsModelClient client = new OptionalClientBuilder().buildCollectionsModelClient();
```

Comment:
change in tests is mostly for changed logic to get the default client name from `listClients` of cadl-dpg (previous default is the whole namespace, now cad-dpg uses last segment of namespace).